frontend/components/navigation/navbar.tsx

```diff
@@ -13,7 +13,7 @@ export default function Navbar() {
 
     return (
         <>
-            <Box as="header" px={4} backgroundColor="white">
+            <Box zIndex={1000} as="header" px={4} backgroundColor="white">
```

Comment:
i don't like 1000, 69 or 420 seem like better choice

Reply:
maybe we should do 69420 instead